tests/tests.js

```diff
@@ -154,7 +154,7 @@ var runTests = function runTests(arrayFrom) {
 			'set': function (x) { throw new Error('setter called: ' + x); }
 		});
 		var myInstance = new MyType();
-		t.throws(function () { myInstance[0] = 'foo'; });
+		t.throws(function () { myInstance.prototype[0] = 'foo'; });
```

Comment:
This doesn't fix it - it throws because `myInstance` isn't a function and doesn't have a `prototype` property - not because the setter is throwing it.

You can verify that by making line 154 throw an `EvalError`, and then pass `EvalError` as the second argument of `t.throws`.